x/collector/keeper/collector.go

```diff
@@ -347,14 +374,19 @@ func (k *Keeper) SetAuctionMappingForApp(ctx sdk.Context, records ...types.Colle
 		if !found1 {
 			return types.ErrorAuctionParmsNotSet
 		}
+
 		var collectorAuctionLookup types.CollectorAuctionLookupTable
+		result1, found := k.GetAuctionMappingForApp(ctx, msg.AppId)
+		if found {
+			collectorAuctionLookup.AssetIdToAuctionLookup = result1.AssetIdToAuctionLookup
```

Comment:
here we are storing auction control table for an AppID and AssetID .
here is how data structure for storing looks like 
type CollectorAuctionLookupTable struct {
AppId                  uint64            
AssetIdToAuctionLookup []AssetIdToAuctionLookupTable
}

request looks like this
collectorTypes.AuctionControlByAppIdProposal{
				Title:       "AddAuctionControl",
				Description: "AddingAuctionControl",
				CollectorAuctionLookupTable: collectorTypes.CollectorAuctionLookupTable{
					AppId: 1,
					AssetIdToAuctionLookup: []collectorTypes.AssetIdToAuctionLookupTable{
						{
							2,
							true,
							true,
							false,
							false,
							1000000,
						},
					},
				},
			},

So if we store for AppId 1 and Asset Id 2 , this gets stored fine .
but after few days , new asset gets added and we need to add this for that also .
If we use same request to store , new one get stored lets say AppId 1 and Asset Id 3 . but instead of appending new asset , this is replacing all previous assets , it will also halt auctions .
So why its replacing ?
new collectorAuctionLookup variable is created but it is not getting existing data but its trying append current asset id app id data .
fix i proposed here is if any assets exist under appid then load them first and append new data to it and store

Reply:
yes, correct 
now it's appending instead of replacing